app/models/concerns/account_settings.rb

```diff
@@ -192,33 +192,36 @@ def reload_library_config
     end
 
     def reload_analytics
+      # rubocop:disable Style/RedundantSelf
       # fall back to the default values if they aren't set in the tenant
-      unless google_analytics_id.present? &&
-             google_oauth_app_name.present? &&
-             google_oauth_app_version.present? &&
-             google_oauth_private_key_secret.present? &&
-             google_oauth_client_email.present? &&
-             (google_oauth_private_key_value.present? || google_oauth_private_key_path.present?)
+      unless self.google_analytics_id.present? &&
```

Comment:
needed to preface everything with self because of the below

``` bash
google_analytics_id => ''
Account.settings['google_analytics_id'] => 'UA-12345'
```